xla/service/gpu/gemm_rewriter.cc

```diff
@@ -53,9 +56,6 @@ limitations under the License.
 #include "xla/stream_executor/device_description.h"
 #include "xla/stream_executor/gpu/gpu_blas_lt.h"
 #include "xla/xla_data.pb.h"
-#include "tsl/platform/errors.h"
```

Comment:
Please avoid rearranging those, we're working on not having this weird ordering, but for now it's required.

Reply:
Sure. I will revert the formatting. For my curiosity, why is this ordering required? When I ran the unit tests with this rerrangement locally, it seemed that it didn't lead to an issue..

Reply:
Reverted.